### DISCUSSION TITLE
Alternative of Team Drive

### DISCUSSION THREAD

**Discussion by mmdhsiam** [anasty17/mirror-leech-telegram-bot#991] [Q&A]
I'm now using  team drive for all files save. But  I have my own edu mail.. Can i use rmy own mail for saving files?  Cz after few days the team drive got vanished.  
if i use own mail  replacing with team drive,  then do i need to use  service account?
If i can use my own edu mail, How is it?

Another thing is, my edu(G suite) mail don't Support creating team drives and  google groups.

**Comment by anasty17** [comment]
Team drive is optional! U can use 'root' or u can put root id or u cqn create folder in root dir and use its id

**Reply by anasty17** [reply]
And sure u should use token.pickle not service accounts

**Reply by mmdhsiam** [reply]
Where i should use token.pickle? Once I upload the token.pickle file while newly creating of fork, where should I use it? I Don't understand 😑

What to do when the used service account  in cloudflare index code? Should I leave it blank or something?  
Also, in index code and gist code Do i select service account is 'false'? 

Since,  i can't creat google group with that mail. Therefore, do you need  Google groups or can you use other Google groups?

**Reply by anasty17** [reply]
Add token.pickle in the root directory or use token_url var.

You can't use Service Accounts while not using TeamDrive for index.

If u r not using service accounts then leave it empty!

No need to google group bcz as i said u can't add sa to root directory. This restriction from edu organisation. And u can't manage from email outside the org.
